lib/install/save.js

```diff
@@ -48,7 +47,7 @@ function saveShrinkwrap (tree, next) {
   if (!npm.config.get('shrinkwrap') || !npm.config.get('package-lock')) {
     return next()
   }
-  createShrinkwrap(tree, {silent: false}, next)
+  require('../shrinkwrap.js').createShrinkwrap(tree, {silent: false}, next)
```

Comment:
This is how I resolved that awful circular require thing I think I mentioned to you a while back :\

Since it's really only ever called once, I didn't bother with the usual conditional require+caching thing.